library/Garden/Schema.php

```diff
@@ -132,6 +146,49 @@ public function dumpSpec() {
     }
 
     /**
+     * Filter fields not in the schema.  The action taken is determined by the configured validation behavior.
+     *
+     * @param array $data
```

Comment:
Methods in the Garden namespace should be fully documented. Add descriptions to the parameters and return.